packages/core/src/utils.ts

```diff
@@ -13,3 +16,25 @@ export function toCommandValue(input: any): string {
   }
   return JSON.stringify(input)
 }
+
+/**
+ *
+ * @param annotationProperties
+ * @returns The command properties to send with the actual annotaiton command
+ * See: https://github.com/actions/runner/blob/ee34f4842e747b452e13235836c92b2bb1606816/src/Runner.Worker/ActionCommandManager.cs#L566
```

Comment:
link to the main branch here so this doesn't go out of date

Reply:
yep I'll update that once the runner PR is merged